deps/pretty-print.lua

```diff
@@ -297,6 +297,26 @@ function dump(value, recurse, nocolor)
   return nocolor and strip(s) or s
 end
 
+local function console_write(fs, s)
+  s = s .. '\n'
+  if uv.guess_handle(uv.fileno(fs))=='tty' then
+    repeat
+      local n, e = uv.try_write(fs, s)
+      if n then
+        s = s:sub(n+1)
+        n = 0
+      else
+        if e:match('^EAGAIN') then
+          n = 0
+        else
+          assert(n, reason)
```

Comment:
Where is `reason` coming from? Should this be `e`?

Reply:
Yes